platforms/database.go

```diff
@@ -5,6 +5,16 @@ import (
 	"strings"
 )
 
+// isKnownOS returns true if the operating system is Linux.
```

Comment:
comment starts with wrong function name; should be "*isLinuxOS* returns true..."

Reply:
@estesp Thansk for reviewing so carefully, I will address these comments in next change : )